pipeline-model-definition/pom.xml

```
@@ -112,6 +112,11 @@
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-cps</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-cps</artifactId>
+      <classifier>tests</classifier>
```

I think this should be in test scope, right? It looks like there is a section below for test scope dependencies, so maybe better to move this down there.

```suggestion
      <classifier>tests</classifier>
      <scope>test</scope>
```